.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+**TL;DR -** [ A short summary of what this PR does and why ]
+
+**What changed?**
+
+- [ More in depth breakdown of changes ]
+- [ Peripheral things that got changed ]
+- [ etc... ]
+
+**Developer Checklist**
+- [ ] Reviewed the [release process](./release_process.md)
+- [ ] Translations up to date
+- [ ] JS minified, SASS compiled
+- [ ] Test suites passing on Jenkins
+- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)
+
+JIRA: [EDUCATOR-XXXX](https://openedx.atlassian.net/browse/EDUCATOR-XXXX)
+
+FIY: @edx/masters-devs-gta
```

Comment:
I admit to forgetting/omitting this tag on occasion. However, I would not check it in because teams/people change.
Same can be said for 'EDUCATOR-' in Jira labels. Different teams use different prefixes. Perhaps just change it to JIRA:(https://openedx.atlassian.net/browse/)

Reply:
My reasoning was to eliminate as much repetitious work as possible. Since, for the immediate future, we own the code, I think it's saving us having to type it. Could be changed when the code owners change

Reply:
I think it might be valid to remove the `EDUCATOR-XXXX` blob though if other folks (e.g. ARCH) touch this

Reply:
Agreed.  Having a ticket skeleton is a Good Idea. I'm of the opinion our fellow committers are  capable in detecting they'll need to use the project prefix for their ticket and not just the ticket number.

And @Andytr1 as we own this repo, we can make updating the template part of our eventual off-boarding of ownership.  Aka, this makes un-owning it an overt act.